src/components/Dropdown/Dropdown.tsx

```diff
@@ -101,9 +116,17 @@ export function Dropdown(props: PropsWithChildren<DropdownProps>): JSX.Element {
     }
 
     if (e.key === 'ArrowDown') {
-      updateFocusedItem(focusedIndex + 1);
+      if (alwaysSelectOption && focusedIndex === items.length-1) {
```

Comment:
if there were only one item in the dropdown, would arrowing down set it as the focused item and the filter would be applied on the next search? from this, it looks like you wouldn't be able to arrow down to set the filter as selected

Reply:
oh whoops I think I was looking at the ArrowUp case instead of ArrowDown. I think the scenario where there's only one option in the dropdown might still be a little weird though. because if the user types something, and then arrows to the dropdown option, I think they wouldn't be able to arrow back to the input where they get what they last typed instead of the dropdown item value. is that the expected behavior?

Reply:
So would we want arrowing up on index 0 to make the value the last inputted value? Or should it work for arrowing down on the last index too

Reply:
I think it should be consistent for both of them, but, I think it would be good to check with product on the behavior they want here. maybe they're fine with not letting people arrow back to their typed text, although I find that a little odd

Reply:
I just looked at the [loom](https://www.loom.com/share/647127a38f0b4cba8e3ae7d0e3b82029) again- in the example, looping back around doesn't actually exist: see time 0:49. Should I ask about that?

Reply:
I looked at the loom, and it seems like the text displayed in the input element shouldn't be updated when the user navigates through the options. The user-typed text always stays there. If we match that behavior, we won't have the problem I brought up about users not being able to go back to their typed text